eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/common/operations/DefaultOperationProcessor.java

```diff
@@ -145,43 +144,26 @@ public void processWithdrawals(
   public void processDepositRequest(
       final MutableBeaconState state, final DepositRequest depositRequest)
       throws BlockProcessingException {
-    final SszList<DepositRequest> depositRequestList =
-        BeaconBlockBodySchemaElectra.required(beaconBlockBodySchema)
-            .getExecutionPayloadSchema()
-            .getDepositRequestsSchemaRequired()
-            .of(depositRequest);
-
-    spec.getBlockProcessor(state.getSlot()).processDepositRequests(state, depositRequestList);
+    spec.getBlockProcessor(state.getSlot()).processDepositRequests(state, List.of(depositRequest));
   }
 
   @Override
   public void processWithdrawalRequest(
       final MutableBeaconState state, final WithdrawalRequest withdrawalRequest)
       throws BlockProcessingException {
-    final SszList<WithdrawalRequest> withdrawalRequests =
-        BeaconBlockBodySchemaElectra.required(beaconBlockBodySchema)
-            .getExecutionPayloadSchema()
-            .getWithdrawalRequestsSchemaRequired()
-            .of(withdrawalRequest);
     final Supplier<ValidatorExitContext> validatorExitContextSupplier =
         spec.atSlot(state.getSlot())
             .beaconStateMutators()
             .createValidatorExitContextSupplier(state);
     spec.getBlockProcessor(state.getSlot())
-        .processWithdrawalRequests(state, withdrawalRequests, validatorExitContextSupplier);
+        .processWithdrawalRequests(state, List.of(withdrawalRequest), validatorExitContextSupplier);
   }
 
   @Override
   public void processConsolidationRequest(
       final MutableBeaconState state, final ConsolidationRequest consolidationRequest)
       throws BlockProcessingException {
-    final SszList<ConsolidationRequest> consolidationRequests =
-        BeaconBlockBodySchemaElectra.required(beaconBlockBodySchema)
-            .getExecutionPayloadSchema()
-            .getConsolidationRequestsSchemaRequired()
-            .of(consolidationRequest);
-
     spec.getBlockProcessor(state.getSlot())
-        .processConsolidationRequests(state, consolidationRequests);
+        .processConsolidationRequests(state, List.of(consolidationRequest));
```

Comment:
it seems like we've hard coded the configuration of needing 1 consolidation request half way through, this could be fun when we need to now handle 2 consolidations, i wonder why we bothered going half way with a list?
If we're hard coding 1 anyway, why do we bother taking a list into the spec function?

Reply:
All EL request operation tests in the reference-tests have two parameters: a pre state and a single operation (ssz encoded) as input. 

With the new `ExecutionRequests` container, the alternative would be to update the reference-tests to use that containers ssz as input (instead of a single EL request). This approach is similar to how some operation tests have the `ExecutionPayload` container ssz as an input. However the current reference-tests are still using the individual operations.

TLDR: reference-tests need to change first, before we can change anything here.

Reply:
> If we're hard coding 1 anyway, why do we bother taking a list into the spec function?
On the spec, each block can have a list of each EL request type, so the spec function does require a list. It is only on the reference-tests that the input is a single operation.

Reply:
the list does make sense it just seems weird to have a single up to a point hten a list, so it'd almost be a better idea to have the list all the way through imo so that when we decide 1 consolidation wasnt enough it's less crazy to fix
but either or, just looked weird.

Reply:
Yeah, did the change it is better :)